common/script/content/achievements.js

```diff
@@ -0,0 +1,163 @@
+import { each } from 'lodash';
+
+let achievementsData = {
+  dilatoryQuest: {
+    icon: 'achievement-dilatory',
+    titleKey: 'achievementDilatory',
```

Comment:
Why not wrap this with the `t` function? https://github.com/HabitRPG/habitrpg/blob/a4fd687510160fc54e5730266f4f921f63f49c53/common/script/content/eggs.js#L6

```js
title: t('achievementDilatory'),
```

Then to render the translation you'd do: `dilatoryQuest.title(language)`